source/reference/ejson/stringify.txt

```diff
@@ -0,0 +1,540 @@
+.. _mongosh-ejson-stringify:
+
+=================
+EJSON.stringify()
+=================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+The ``EJSON.stringify()`` method converts BSON values to strings. 
+
+Syntax
+------
+
+The ``EJSON.stringify()`` method takes a BSON object as input and
+optional modifiers that control the format of the output string.
+
+.. code-block:: javascript
+   :copyable: false
+
+   EJSON.stringify(BSON object, [replacer], [space], [options])
+
+Command Fields
+--------------
+
+The ``EJSON.stringify()`` method has these fields:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Field
+     - Type
+     - Necessity
+     - Description
+
+   * - ``value``
+     -  BSON object
+     -  Required
+     -  The object ``EJSON.stringify`` transforms
+
+   * - ``replacer``
+     -  array or function
+     -  Optional
+     -  Modifies output. If the value exists but it is not an array or a
+        function, ``EJSON.stringify`` returns all document fields. 
+ 
+        .. list-table::
+           :header-rows: 1
+           :widths: 30 70
+
+           * - Value
+             - Effect
+
+           * - array
+             - An array of document fields to include in the output
+
+           * - function
+             - A function that takes two parameters, ``key`` and
+               ``value``. ``key`` provides the function's ``this``
+               context. EJSON returns the transformed ``value``. See
+               below for an :ref:`example <ex-ejson-function>`   
+
+   * - ``spacer``
+     -  integer or string
+     -  Optional
+     -  Controls spacing in the output. Use ``null`` as a place holder
+        for ``replacer`` if you only want to specify the ``spacer``
+        option.
+ 
+        .. list-table::
+           :header-rows: 1
+           :widths: 30 70
+
+           * - Value
+             - Effect
+
+           * - integer
+             - The number of spaces to indent each level. ``10`` is the
+               maximum.
+
+           * - string
+             - The character to use to indent each level.
+
+   * - ``options``
+     -  boolean
+     -  Optional
+     -  Additional configuration options
+      
+        .. list-table::
+           :header-rows: 1
+           :widths: 25 25 50
+
+           * - Option
+             - Default
+             - Meaning
+
+           * - ``legacy``
+             - ``false``
+             - Use the Extended JSON v1 specification for output.
+
+           * - ``relaxed``
+             - ``false``
+             - Enable relaxed mode for Extended JSON. Returns native 
+               JavaScript types instead of BSON types when possible.
```

Comment:
```suggestion
             - ``true``
             - Enable relaxed mode for Extended JSON. Returns native 
               JSON types instead of attaching BSON type information where applicable.
```